azure-pipelines.yml

```diff
@@ -54,3 +57,20 @@ steps:
   env: {
     testEnv: 'ci'
   }
+- script: |
+    git config user.email builds@sitecore.com
+    git config user.name "Automated Build"
+    git remote set-url origin https://$GITHUB_PAT_ENV@github.com/Sitecore/jss.git
+    git checkout $(Build.SourceBranchName)
+    $(npm bin)/lerna version prerelease --preid canary --force-publish --message "version %s [skip ci]" --yes
```

Comment:
What's the significance of "[skip ci]" here?

Reply:
Only documented in this [commit message](https://github.com/Sitecore/jss/pull/513/commits/b302e6756618ee6f204773afb445303c86ebd48b) but it [prevents Azure DevOps from running a build](https://docs.microsoft.com/en-us/azure/devops/pipelines/repos/azure-repos-git?view=azure-devops&tabs=yaml#skipping-ci-for-individual-commits) for the commit (and thus triggering the version again).

I'll add a comment to the pipeline yml.

Reply:
Added a bunch of comments :)